src/content/en/updates/2017/02/navigation-preload.md

```diff
@@ -0,0 +1,338 @@
+responding using the network…
+
+{% framebox height="38px" %}
+<style>
+  .group {
+    height: 38px;
+    display: flex;
+  }
+  .group > div {
+    display: flex;
+    align-items: center;
+    font: normal 1.2rem/1 sans-serif;
+    text-shadow: 0 1.6px 2.7px rgba(0,0,0,0.5);
+    color: #fff;
+    box-sizing: border-box;
+    padding: 0 8px;
+    white-space: pre;
+  }
+  .sw-startup {
+    margin-right: 1px;
+    width: 100px;
+    background: #f0c457;
+  }
+  .network-request {
+    flex: 1;
+    background: #6ea1e3;
+  }
+</style>
+<div class="group">
+  <div class="sw-startup">SW boot</div>
+  <div class="network-request">Network request</div>
+</div>
+{% endframebox %}
+
+…the network request is delayed by the service worker booting-up.
+
+We're continuing to reduce the boot-up time by [using code-caching in
+V8](http://v8project.blogspot.com/2015/07/code-caching.html), by [skipping
+service workers that don't have a fetch
+event](https://bugs.chromium.org/p/chromium/issues/detail?id=605844), by
+[launching serivce workers
+speculatively](https://codereview.chromium.org/2045153003), and other
+optimisations, but it'll always be greater than zero.
+
+Facebook brought the impact of this problem to our attention, and asked for a
+way to perform the network request in parallel:
+
+{% framebox height="77px" %}
+<style>
+  .group {
+    height: 38px;
+    display: flex;
+  }
+  .group > div {
+    display: flex;
+    align-items: center;
+    font: normal 1.2rem/1 sans-serif;
+    text-shadow: 0 1.6px 2.7px rgba(0,0,0,0.5);
+    color: #fff;
+    box-sizing: border-box;
+    padding: 0 8px;
+    white-space: pre;
+  }
+  .sw-startup {
+    margin-right: 1px;
+    width: 100px;
+    background: #f0c457;
+  }
+  .network-request {
+    flex: 1;
+    transform: translate(-101px, 1px) translate(0, 100%);
+    background: #6ea1e3;
+  }
+</style>
+<div class="group">
+  <div class="sw-startup">SW boot</div>
+  <div class="network-request">Network request</div>
+</div>
+{% endframebox %}
+
+…and we said "yeah, seems fair".
+
+## "Navigation preload" to the rescue {: #the-solution }
+
+Navigation preload is a new experimental feature that lets you say "Hey, when
+the user makes a GET navigation request, start making the network request while
+the service worker is booting up.
+
+The startup delay is still there, but it doesn't block the network request, so
+the user gets content sooner.
+
+Here's a video of it in action, where the service worker is given a deliberate
+500ms startup delay using a while-loop:
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="icv_DpQLryE"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+[Here's the demo
+itself](https://jakearchibald.github.io/isserviceworkerready/demos/nav-preload/).
+To get the benefits of navigation preload, you'll need [Chrome
+Canary](https://www.google.com/chrome/browser/canary.html) with
```

Comment:
Chrome 57 or Chrome 57 Canary. Always use a specific version number so that this document has proper context when someone stumbles on it a  year from now.